README.md

```diff
@@ -1,65 +1,160 @@
+
+![Actions](TA-puppet-alert-orchestrator/readme/img/orchestrator_actions.png)
+
+#### Splunk Cloud
+
+The Puppet Enteprise Console URL and Orchestration Services port (**default**: `8143`) need to be added to the [outbound port configuration](https://docs.splunk.com/Documentation/SplunkCloud/latest/Config/ConfigureOutboundPorts). If this is not completed, you will see the following error in the `splunkd` log:
+
+```
+ERROR ExecProcessor [28983 ExecProcessor] - message from “/opt/splunk/bin/python3.7 /opt/splunk/etc/apps/TA-puppet-alert-orchestrator/bin/inputs/orchestrator_actions.py” TA-puppet-alert-orchestrator: Failed to build event message - HTTPSConnectionPool(host=‘PE_CONSOLE_URL’, port=8143): Max retries exceeded with url: /orchestrator/v1/tasks (Caused by NewConnectionError(‘<urllib3.connection.HTTPSConnection object at 0x7f30adc1df10>: Failed to establish a new connection: [Errno 110] Connection timed out’))
+```
+
+**Notes**:
+
+> * By default the custom input script only checks the `production` environment. As a user can override the default environment when configuring the action, they can also input a custom Plan or Task name that may not be listed in the default environment.
+
+> * This add-on has **only** been tested on the [Victoria Experience](https://docs.splunk.com/Documentation/SplunkCloud/latest/Admin/Experience) of Splunk Cloud. To utilize this add-on with the Classic Experience, your installation will need to include an [Inputs Data Manager (IDM)](https://docs.splunk.com/Documentation/SplunkCloud/latest/Admin/Intro#Splunk_Cloud_Platform_features).
 
+#### Splunk Enterprise
+
+The interval (**default**: `60m`) at which the input script runs is configurable **only** within Splunk Enterprise by adding the following to a local `inputs.conf` file for the add-on:
+
+```
+[script://./bin/inputs/orchestrator_actions.py]
+# 12h interval
+interval = 43200
+# To specify a cron schedule, use the following format:
+# "<minute> <hour> <day of month> <month> <day of week>"
+```
 
 ## Usage
 
-Once the add-on has been properly configured you can begin creating new Alerts to trigger specified actions (e.g. Run a Task, Generate a Detailed Report) when a new event matching the Alert is received by Splunk.
+Once the add-on has been properly configured you can begin creating new Alerts to trigger specified actions (e.g. Run a Puppet Plan, Run a Puppet Task, Generate a detailed Puppet report) when a new event matching the Alert is received by Splunk.
 
 1. Create a Splunk search:
 
-	![Splunk Search](TA-puppet-alert-actions/README/pics/splunk_search.png)
+	![Splunk Search](TA-puppet-alert-orchestrator/readme/img/splunk_search.png)
 	
 2. Save the search as an Alert:
 
-	![Save Splunk Search](TA-puppet-alert-actions/README/pics/save_as_alert.png)
+	![Save Splunk Search](TA-puppet-alert-orchestrator/readme/img/save_as_alert.png)
 
 3. Configure the Alert Actions:
```

Comment:
not sure, but should this get a rename?

Reply:
Splunk naming conventions suggest putting any doc images in `readme/img`. They were previously split between `README/img` and `README/pics`.